operator_ui/src/components/JobRuns/TaskExpansionPanel.tsx

```diff
@@ -22,100 +27,137 @@ const fontStyles = () =>
   })
 
 interface IItemProps extends WithStyles<typeof fontStyles> {
-  keyOne: string
-  valOne: string
-  keyTwo: string
-  valTwo: string
+  colATitle: string
+  colAValue: string
+  colBTitle: string
+  colBValue?: string | null
 }
 
 const Item = withStyles(fontStyles)(
-  ({ keyOne, valOne, keyTwo, valTwo, classes }: IItemProps) => (
+  ({ colATitle, colAValue, colBTitle, colBValue, classes }: IItemProps) => (
     <Grid container>
       <Grid item sm={2}>
-        <p className={classes.header}>{keyOne}</p>
-        <p className={classes.subHeader}>{valOne || 'No Value Available'}</p>
+        <p className={classes.header}>{colATitle}</p>
+        <p className={classes.subHeader}>{colAValue}</p>
```

Comment:
dont remember exactly what's going on here conceptually, but why isn't this `nullable` as the other col?

Reply:
@rupurt reminder about this